src/Plot/Matplotlib.luna

```diff
@@ -58,6 +58,39 @@ class Matplotlib:
         result = self.callHandlingError "getPngBase64" CString []
         result.toText
 
+    # Saves the chart to an image file.
+    #
+    # Image will have dimensions as requested in function arguments. 
+    # The image file format is deduced from filename extension.
+    # Supported formats: eps, pdf, pgf, png, ps, raw, rgba, svg, svgz.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # >
+    # > def main:
+    # >     l1 = [1,2,3,4,5]
+    # >     l2 = [11,12,13,14,15]
+    # >     col1 = Column.fromList "col1" Int64Type l1
+    # >     col2 = Column.fromList "col2" Int64Type l2
+    # >     table = Table.fromColumns [col1 , col2]
+    # >     plt = table.plot "col1" "col2" . write 800 600 "foo.png"
+    # >     None
+    #
+    # errors:
+    # An exception is raised when output file is not writable.
+    # An exception is raised when output file extension is not of one of supported image formats.
+    # An exception is raised when requested image width or height is less or equal to zero.
+    #
+    # `width`: Requested width of the image, must be greater than zero.
+    # `height`: Requested height of the image, must be greater than zero.
+    # `filename`: Output path where image will be stored, must be writable.
+    #
+    # `return`: None
```

Comment:
I think we don't need documentation here, as this class is not really part of public interface. Nor the code sample actually calls it.

Reply:
I think that this snippet was intended for the function added in the `Chart` class.